cryptofeed/exchange/okcoin.py

```diff
@@ -204,7 +242,30 @@ async def message_handler(self, msg: str, conn, timestamp: float):
                 await self._book(msg, timestamp)
             elif 'swap/funding_rate' in msg['table']:
                 await self._funding(msg, timestamp)
+            elif 'spot/order' in msg['table']:
+                await self._order(msg, timestamp)
             else:
                 LOG.warning("%s: Unhandled message %s", self.id, msg)
         else:
             LOG.warning("%s: Unhandled message %s", self.id, msg)
+    
+    def connect(self) -> List[Tuple[AsyncConnection, Callable[[None], None], Callable[[str, float], None]]]:
+        ret = []
+        for channel in self.subscription or self.channels:
+            if is_authenticated_channel(channel):
+                ret.append((AsyncConnection(self.address, self.id, **self.ws_defaults), self.auth_subscribe, self.message_handler))
+            else:
+                ret.append((AsyncConnection(self.address, self.id, **self.ws_defaults), self.subscribe, self.message_handler))
+
+        return ret
+    
+    async def auth_subscribe(self, conn: AsyncConnection, options=None):
+        self.__reset()
+        timestamp, sign = generate_token(self.key_id, self.key_secret)            
+        login_param = {"op": "login", "args": [self.key_id, self.config.okex.key_passphrase, timestamp, sign.decode("utf-8")]}
+        login_str = json.dumps(login_param)
+        await conn.send(login_str)
+        await asyncio.sleep(5)
+        sub_param = {"op": "subscribe", "args": ["spot/order:BTC-USDT"]}
```

Comment:
this is hard coded to only BTC, right, not what the user has configured?

Reply:
updated and improved in second commit - thanks for the comments and suggestions